### ISSUE TITLE
Fails to build with 'PYTHONQTALL_CONFIG=PythonQtCore'

### ISSUE THREAD

**Issue by iakov** [MeVisLab/pythonqt#200]
Generated files for core component (`com_trolltech_qt_core/` folder) contain  `#include "qwidget.h"` and other similar includes that are not part of the `core` module.
Probably, this is an issue with `typsystem_core.xml`, that has type declarations for `QTextDocument` and other types/enums from `QtGui`.

**Comment by usiems** [comment]
Yes, it seems the include of QTextDocument is necessary because of Qt namespace functions convertFromPlainText and mightBeRichText. These would need to be moved to a different wrapper object to fix this. This can't be easily accomplished in a generic manner, if I remember the parser code correctly. (Everything is parsed in one go, and everything from the same namespace is pooled into one object, without consideration for the original Qt package.)
I might be wrong, though.
